_observing-your-data/alerting/dashboards-alerting.md

```diff
@@ -9,84 +9,297 @@ nav_order: 50
+
+**High CPU usage alert**
+```json
+{
+  "name": "High CPU Usage Alert",
+  "description": "Alerts when the average CPU usage exceeds 80% in the last hour",
+  "schedule": {
+    "interval": {
+      "period": 1,
+      "unit": "HOURS"
+    }
+  },
+  "trigger": {
+    "condition": {
+      "script": {
+        "source": "
+          double avgCpuUsage = ctx.payload.aggregations.avg_cpu_usage.value;
+          return avgCpuUsage > 80;
+        "
+      }
+    }
+  },
+  "input": {
+    "search": {
+      "request": {
+        "indices": ["system-metrics"],
+        "body": {
+          "size": 0,
+          "aggs": {
+            "avg_cpu_usage": {
+              "avg": {
+                "field": "cpu_usage"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "actions": [
+    {
+      "name": "Notify on Slack",
+      "type": "slack",
+      "slack": {
+        "message": "High CPU usage detected: {{ctx.payload.aggregations.avg_cpu_usage.value}}%"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+**Low disk space alert**
+```json
+{
+  "name": "Low Disk Space Alert",
+  "description": "Alerts when the available disk space falls below 20%",
+  "schedule": {
+    "interval": {
+      "period": 30,
+      "unit": "MINUTES"
+    }
+  },
+  "trigger": {
+    "condition": {
+      "script": {
+        "source": "
+          double diskUsagePercentage = ctx.payload.aggregations.disk_usage_percentage.value;
+          return diskUsagePercentage > 80;
+        "
+      }
+    }
+  },
+  "input": {
+    "search": {
+      "request": {
+        "indices": ["system-metrics"],
+        "body": {
+          "size": 0,
+          "aggs": {
+            "disk_usage_percentage": {
+              "script": {
+                "source": "
+                  double totalDiskSpace = doc['disk_total'].value;
+                  double usedDiskSpace = doc['disk_used'].value;
+                  return (usedDiskSpace / totalDiskSpace) * 100;
+                "
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "actions": [
+    {
+      "name": "Notify on Email",
+      "type": "email",
+      "email": {
+        "to": ["ops@example.com"],
+        "subject": "Low Disk Space Alert",
+        "message": "Available disk space is below 20%: {{ctx.payload.aggregations.disk_usage_percentage.value}}%"
+      }
+    }
+  ]
+}
+```
+{% include copy-curl.html %}
+
+## Viewing alerting events
+
+To view alerting events, follow these steps:
+
+1. Open your local instance of OpenSearch Dashboards. This is typically `http://localhost:5601/app/home#/`.
+2. Navigate to **Alerting** from the main menu. You should see a list of all the alerting monitors you have configured. Select the monitor for which you want to view alerts.
```

Comment:
Minor issue, but the homepage for the Alerting plugin shows a list of alerts across all monitors. On that page, there's a tab that will redirect to a list of monitors.

Here's a screenshot of the homepage for reference.
![Screenshot 2024-05-29 at 11 28 19 AM](https://github.com/opensearch-project/documentation-website/assets/79280347/cdc2f94f-ad98-4b33-a4af-c9b859012e28)